store/index.js

```diff
@@ -196,27 +196,28 @@ export const actions = {
       await dispatch('reset', activeNetwork);
     }
 
-    const loadFromKetherView = activeNetwork == 'rinkeby'; // Only deployed KetherView on rinkeby
+    const loadFromKetherView = !!state.networkConfig.ketherViewAddr; // Only use View contract if deployed
     const loadFromEvents = true; // Okay to do it always? or should we do: state.loadedBlockNumber > 0
 
     if (loadFromKetherView) {
-      const limit = 200;
+      const limit = 420; // 4 queries to load 1621 ads on mainnet
       const len = await contract.getAdsLength();
       commit('setAdsLength', len);
       //TODO test off-by-ones
       for (let offset = 0; offset < len; offset+=limit) {
-        const ads = await ketherView.allAds(contract.address, ketherNFT.address, offset, limit);
         // TODO: the structure here contains a `wrapped` bool which we can use to power proxying publish to the nft contract and wrapping/unwrapping.
```

Comment:
@mveytsman What's this todo for? I don't quite get the action item

Reply:
It was to build the publishing/wrapping stuff which I then did, so you can delete it :)